eos-toolbox-dev-depends

```diff
@@ -0,0 +1,74 @@
+# The eos-toolbox-dev package is intended to be included in the eos-toolbox
+# container images as a way to provide the base development packages needed
+# for common development.
+#
+# Packages listed here should be commonly used, of high value for
+# development, and expected to be relevant to a whole group of Endless
+# developers.
+#
+# Some packages included in the eos-dev package (e.g. gnome dev packages)
+# are not included here on purpose given they increase the container image
+# size quite substantially. Developers that need the extra packages can
+# still install them manually using apt-get from an eos-toolbox container.
+
+acl
+apt-file
+apt-transport-https
```

Comment:
```
wjt@tensionsheet-redux:~$ apt show apt-transport-https
...
Description: transitional package for https support
 This is a dummy transitional package - https support has been moved into
 the apt package in 1.5. It can be safely removed.
```

```suggestion
```

Reply:
ack, will remove it